config/nav.yml

```diff
@@ -50,6 +47,10 @@ nav:
         - Installing kn: install/client/install-kn.md
         - Customizing kn: install/client/configure-kn.md
         - kn plugins: install/client/kn-plugins.md
+      # Istio Installation
+      - Installing Istio for Knative: install/installing-istio.md
+      # Cert-manager Installation
+      - Installing cert-manager: install/installing-cert-manager.md
```

Comment:
Would it make sense to keep these under the "install with YAML" section, but maybe in an "advanced install" group? Part of me likes the idea of the main left sidebar groups just being the different install options (yaml / operator / offering) and upgrading / installing.

Reply:
I'm not a fan of having the Istio installation under install with YAML because it's also needed for the Operator installation if the user chooses to use Istio.

How about nesting them under "Install advanced options" section? See the [new preview](https://deploy-preview-4690--knative.netlify.app/development/install/).